Source/Scene/ClassificationModel.js

```diff
@@ -0,0 +1,1869 @@
+define([
+        '../Core/arraySlice',
+        '../Core/BoundingSphere',
+        '../Core/Cartesian3',
+        '../Core/Cartesian4',
+        '../Core/Cartographic',
+        '../Core/Color',
+        '../Core/combine',
+        '../Core/ComponentDatatype',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/DistanceDisplayCondition',
+        '../Core/FeatureDetection',
+        '../Core/getBaseUri',
+        '../Core/IndexDatatype',
+        '../Core/joinUrls',
+        '../Core/loadArrayBuffer',
+        '../Core/Matrix2',
+        '../Core/Matrix3',
+        '../Core/Matrix4',
+        '../Core/PrimitiveType',
+        '../Core/Quaternion',
+        '../Core/Queue',
+        '../Core/RuntimeError',
+        '../Core/Transforms',
+        '../Core/WebGLConstants',
+        '../Renderer/ShaderSource',
+        '../ThirdParty/GltfPipeline/ForEach',
+        '../ThirdParty/GltfPipeline/getAccessorByteStride',
+        '../ThirdParty/GltfPipeline/numberOfComponentsForType',
+        '../ThirdParty/GltfPipeline/parseBinaryGltf',
+        '../ThirdParty/when',
+        './AttributeType',
+        './Axis',
+        './ClassificationType',
+        './getAttributeOrUniformBySemantic',
+        './HeightReference',
+        './ModelMaterial',
+        './ModelMesh',
+        './ModelNode',
+        './SceneMode',
+        './Vector3DTileBatch',
+        './Vector3DTilePrimitive'
+    ], function(
+        arraySlice,
```

Comment:
Did you remove unused defines after the latest refactor?

Reply:
Removed.